src/DynamoCoreWpf/ViewModels/Search/SelectionNavigator.cs

```diff
@@ -70,6 +75,27 @@ internal void MoveSelection(NavigationDirection direction)
             if (root == null || !root.Any())
                 return;
 
+            // Selection can be null, if user just searched and currently selected element is top result.
+            if (selection == null)
```

Comment:
Here we should check `selection == null`, not `selection == topResult` because selection in fact can be top result.